StatCounter.py

```diff
@@ -246,10 +241,13 @@
 
 
 #write teammates and encounter matrix to file
-pickle.dump(teammateMatrix,teammatefile)
-teammatefile.close()
-pickle.dump(encounterMatrix,encounterfile)
-encounterfile.close()
+filename="Raw/moveset/"+tier+"/teammate"+specs+".pickle"
+teammatefile=open(filename,'wb')
+with open(filename, 'wb') as teammate_file:
+    pickle.dump(teammateMatrix, teammate_file)
+filename="Raw/moveset/"+tier+"/encounterMatrix"+specs+".pickle"
+with open(filename, 'wb') as encounter_file:
+    pickle.dump(encounterMatrix, encounter_file)
```

Comment:
OK, fine, but not actually a change that is relevant to converting to Python 3

Reply:
The file context stuff isn't immediately relevant, but pickle files do have to be written to binary; before, they were just written as ASCII files I believe.

Reply:
Ah OK. Fine as-is